origami.json

```diff
@@ -40,6 +40,12 @@
 			"template": "demos/src/simple-footer.mustache",
 			"expanded": true,
 			"description": ""
+		},
+		{
+			"name": "pa11y",
+			"template": "demos/src/pa11y.mustache",
+			"expanded": true,
```

Comment:
sure! I wrote this test few weeks ago and I didn't realise that it had the old format, thanks!